consts/validation.js

```diff
@@ -13,7 +13,7 @@ const regex = {
 
 const enums = {
   LANG_ENUM: ['en', 'ua'],
-  LANG_LEVEL_ENUM: ['Beginner', 'Pre-Intermediate', 'Intermediate', 'Upper-Intermediate', 'Advanced', 'Native'],
+  SUBJECT_LEVEL_ENUM: ['Beginner', 'Intermediate', 'Advanced', 'Test Preparation', 'Professional', 'Specialized'],
```

Comment:
that naming has been approved? i thought it was something to consider

Reply:
not at 100%, but I think it wouldn't be a problem to make small changes it if needed

Reply:
what does specialized mean?
also i thought we discussed something like beginner | inter | adv | higher, didn't we?

Reply:
i guess you are right, it will be better like: ['Beginner', 'Intermediate', 'Advanced', 'Test Preparation', 'Professional] without Specialized